### DISCUSSION TITLE
how to get user email with invite url ?

### DISCUSSION THREAD

**Discussion by NMTuan** [directus/directus#16134] [Archive]
I use `users.invites.send()` send the invite mail, and get the invite url,

now, how to get the user email with the invite url `?token=xxx` in js-sdk ?

**Comment by br41nslug** [comment]
Im not sure im following what you're trying to do, you want to include your account token in an invite link to new users? as an access token is retrieved by logging in this cannot be done yet for not activated accounts. Besides that tokens are temporary and expire after 15 minutes by default (unless they're static tokens but you really don't want to be sending those over insecure media).

**Reply by NMTuan** [reply]
I want show email filed in invite accept page

**Reply by br41nslug** [reply]
Can you elaborate? Because the email is already filled on the accept page?
![image](https://user-images.githubusercontent.com/9389634/197485057-d42add46-2459-476c-95f1-84135d587e78.png)

**Reply by NMTuan** [reply]
I want to show on my own page

**Reply by NMTuan** [reply]
invite page http://localhost:3000/invite
![image](https://user-images.githubusercontent.com/2149763/197656749-cfca5719-c8be-40b4-925d-de5326dfdd93.png)
accept page http://localhost:3000/invite?token=test_token
![image](https://user-images.githubusercontent.com/2149763/197656806-c05c57c5-78db-423a-802e-06c4c7e26b5a.png)

the form show invite_token and password fields.
I want show the email and password fields.

**Reply by br41nslug** [reply]
If you want to pre-fill the email address there then wouldn't you rather use the email as parameter than an arbitrary access token?

**Reply by reckseba** [reply]
The token is a jwt. You can decrypt it with https://www.npmjs.com/package/jwt-decode for example.

`import jwt_decode from "jwt-decode";
const decoded = jwt_decode(token);
const email = decoded.email;
`
